android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetActivity.java

```diff
@@ -283,6 +291,14 @@ protected void onUserLeaveHint() {
         }
     }
 
+    /**
+     * Implementation of the {@code PermissionAwareActivity} interface.
+     */
+    @Override
+    public void requestPermissions(String[] permissions, int requestCode, PermissionListener listener) {
```

Comment:
where is this method called from ?

Reply:
oh now I see that PermissionAwareActivity comes from react

Reply:
Yep, it's part of the interface.